lib/packwerk/package.rb

```diff
@@ -30,7 +30,12 @@ def initialize(name:, config:)
 
     sig { returns(T::Boolean) }
     def enforce_dependencies?
-      @config["enforce_dependencies"] == true
+      [true, "strict"].include?(@config["enforce_dependencies"])
+    end
+
+    sig { returns(T::Boolean) }
+    def strictly_enforce_dependencies?
```

Comment:
Is this used?

Reply:
Nope! An artifact of a spike that I didn't end up using, removing.